source/_integrations/smtp.markdown

```diff
@@ -109,9 +109,9 @@ burglar:
 
 The optional `target` field is used to specify recipient(s) for this specific service call. When `target` field is not used, this message will be sent to default recipient(s), in this example, my_intruder_alert@example.com.
 
-The optional `images` field adds in-line image attachments to the email. This sends a text/HTML multi-part message instead of the plain text default.
+The optional `html` field makes a custom text/HTML multi-part message, allowing total freedom for sending rich html emails by defining the html content. In them, if you need to include images, you can pass both arguments (`html` and `images`), the images will be joined with the basename of the images, so they can be included in the html page with `src="cid:image_name.ext"`.
```

Comment:
Possibly swap the word 'joined' for 'attached'.

Reply:
Done